hack/update-hack-tools.sh

```diff
@@ -35,9 +35,12 @@ kube::golang::verify_go_version
 pushd "${KUBE_ROOT}/hack/tools" >/dev/null
   echo "=== tidying go.mod/go.sum in hack/tools"
   go mod edit -fmt
-  go mod download
+  go mod vendor
   go mod tidy
```

Comment:
not sure it matters, but tidy before vendor?

Reply:
ack switching it around